### PR TITLE
gather_method = "max" or "mean" of frequencies in a band for spectral…

### PR DESCRIPTION
**Pull request by davidmeunier79**
… connectivity

### CODE REVIEW

**Comment by davidmeunier79** [comment]
@jasmainak I am afraid the errors I have now starting from your backup are not related to my modifications

File: ephypype/spectral.py [hunk 1]
```diff
@@ -23,16 +29,28 @@ def compute_spectral_connectivity(data, con_method, sfreq, fmin, fmax,
...
+                mt_adaptive=False, n_jobs=1)
+            con_matrix = np.amax(con_matrix, axis=2)
+
+        elif gathering_method == "none":
+            con_matrix, _, _, _, _ = spectral_connectivity(
+                data, method=con_method, sfreq=sfreq, fmin=fmin,
+                fmax=fmax, faverage=False, tmin=None, mode='multitaper',
+                mt_adaptive=False, n_jobs=1)
+        else:
+            print('Unknown gathering method')
```
Comment (jasmainak) [inline ephypype/spectral.py]: You should raise a `ValueError` instead of printing ...

**Comment by jasmainak** [comment]
@davidmeunier79 yes the errors are not related to your pull request. We should fix them separately

**Comment by jasmainak** [comment]
@davidmeunier79 almost clean ...

Your first commit is a "merge commit" though. See my explanation in the other pull request.

**Comment by davidmeunier79** [comment]
Please provide me with the specific instructions because Im lost... Everytime I try to to a rebase, either it is failing ("patch cannot be applied") or I end up with a "merge branch" commit message. I may be easy once you have done it but it is not intuitive. I have tried to follow what is explained in this tutorial but it ended up as everything was on my master branch, and from what you told earlier, I should not try a pull request from davidmeunier79:master ->neuropycon:master, so I quit. Can you provide me (by e-mail if you prefer) the list of commands? Many thanks

**Comment by annapasca** [comment]
Hi @davidmeunier79 for me to rebase a PR it was useful this link 
https://github.com/edx/edx-platform/wiki/How-to-Rebase-a-Pull-Request

I hope it helps

**Comment by jasmainak** [comment]
@davidmeunier79 it looks like your branch is a mess. I tried rebasing but I don't know the code well enough to help. The easiest solution at this point might be to start a new branch from master and copy the edits onto this new branch ...

**Comment by davidmeunier79** [comment]
> @davidmeunier79 it looks like your branch is a mess. I tried rebasing but I don't know the code well enough to help. The easiest solution at this point might be to start a new branch from master and copy the edits onto this new branch ...

Yes this what I try to do since the beginning. the problem is reated to stuff I did earlier that you coming. And in between there has been some modifications over the same files I had previously modified. Also, my problem is related to MY branch master, because everytime I try a rebase or merge, I end up with the fusion problem related to the modifications done earlier in my master branch. Maybe there is a way to loose these modifications?

**Comment by jasmainak** [comment]
okay so you should delete your master branch and get a new one.

```sh
$ git branch -D master
$ git fetch upstream master:master
```

**Pull request closed without merging**